lib/src/transformer/impl_generation.dart

```diff
@@ -364,19 +364,20 @@ class ImplGenerator {
 
     typedMap.node.members
         .where((member) => member is FieldDeclaration)
-        .where((FieldDeclaration member) => !member.isStatic)
-        .forEach((FieldDeclaration field) {
+        .where((member) => !(member as FieldDeclaration).isStatic) // ignore: avoid_as
+        .forEach((field) {
```

Comment:
#nit Just so there is not a bunch of changes could the param be prefixed with an underscore and the new var not?

```dart
.forEach((_field) {
  final field = _field as FieldDeclaration;
})
```